orange/src/main/java/com/hartwig/hmftools/orange/conversion/ConversionUtil.java

```diff
@@ -0,0 +1,28 @@
+package com.hartwig.hmftools.orange.conversion;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Objects;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public class ConversionUtil {
+
+    private ConversionUtil() {
+    }
+
+    @NotNull
+    public static <T, R> Iterable<R> mapToIterable(@Nullable Collection<T> collection, Function<T, R> mapper) {
+        var nonNull = Objects.requireNonNullElseGet(collection, List::<T>of);
```

Comment:
`var`? not sure that hmftools can be released for Java 17.

Reply:
`var` was introduced in java 10. I'll remove my uses of var though, since it's not used anywhere else in the codebase afaict.